src/main/java/edu/ucla/library/bucketeer/handlers/LoadCsvHandler.java

```diff
@@ -97,7 +97,8 @@ public void handle(final RoutingContext aContext) {
         final HttpServerRequest request = aContext.request();
         final MultiMap formAttributes = request.formAttributes();
         final Set<FileUpload> csvUploads = aContext.fileUploads();
-        final String slackHandle = StringUtils.trimToNull(formAttributes.get(Constants.SLACK_HANDLE));
+        final String slackHandle = StringUtils.trimToNull(formAttributes.get(Constants.SLACK_HANDLE)).replace(
```

Comment:
trimToNull() can return a null, which will give you a NullPointerException on your .replace(). The replace should go down in the `else` where we know the value isn't null.